tests/installation/installation_overview_before.pm

```diff
@@ -21,7 +21,11 @@ sub run() {
 
     # overview-generation
     # this is almost impossible to check for real
-    assert_screen "inst-overview";
+    # See poo#12322. Prevent checks before overview is fully loaded
+    # 'inst-overview' needle is used in many places and sometimes includes only
+    # parts which are there while overview is still loading. This check has to be
+    # performed only once, as state of buttons can be different
```

Comment:
Here is good [example](https://openqa.suse.de/tests/1010304#step/installation_overview/1) This is what happens on sle, and what happens next is that we continue executing tests, even though we are in "Analyzing your system" state. As mentioned above, this issue is only with sle in GUI mode, as for text mode we also match "install" button and it doesn't change it's state while navigating in menu.